.github/workflows/ci.yaml

```diff
@@ -697,21 +697,31 @@ jobs:
         run: photofinish run healthy-27-node-SAP-cluster -u "http://$TRENTO_DEMO_IP/api/collect" "$TRENTO_API_KEY"
 
   obs-commit:
-    name: Commit the project on OBS
+    name: Commit to OBS to generate container image and RPM
     runs-on: ubuntu-20.04
     if: github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
     needs: [static-code-analysis, test, test-fe, test-e2e]
+    strategy:
```

Comment:
I'm not sure if we should use the same strategy. 
In the rpm package we should leverage the `_service` file usage, which makes the process a bit different